src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -176,32 +169,39 @@ export const Sidebar: React.FC<Props> = ({
     }
   }, [redeemCode, campaignData, refetchAll])
 
-  const discountText = getDiscountText(textKeys)(
-    redeemedCampaigns,
-    offerData.cost.monthlyGross.currency,
-  )
-
   const isNorwegianBundle = isBundle(offerData) && isNorwegian(offerData)
+  const discountText = isNorwegianBundle
+    ? textKeys.SIDEBAR_NO_BUNDLE_DISCOUNT_TEXT()
+    : getDiscountText(textKeys)(
+        redeemedCampaigns,
+        offerData.cost.monthlyGross.currency,
+      )
+  const clearDiscountCode = () => {
+    removeDiscountCode()
+      .then(() => {
+        const cookieStorage = new CookieStorage()
+        cookieStorage.setItem('_hvcode', '', {
+          path: '/',
+        })
+      })
+      .then(() => refetchAll())
+  }
+
+  const hasRedeemedCampaigns = Boolean(redeemedCampaigns.length)
```

Comment:
Why not write this more explicitly?

```suggestion
  const hasRedeemedCampaigns = redeemedCampaigns.length > 0
```